docs/src/routes/+layout.svelte

```
@@ -1,19 +1,78 @@
+            offers: {
+                '@type': 'Offer',
+                price: '0',
+                priceCurrency: 'USD'
+            },
+            aggregateRating: {
+                '@type': 'AggregateRating',
+                ratingValue: '5',
+                bestRating: '5',
+                ratingCount: String(githubStats.stars)
+            }
+        })}<\/script>`
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)